### ISSUE TITLE
Where's GeoXplore for NN?

### ISSUE THREAD

**Issue by tachyon-work** [gtkfi/ArcSDM#124]
Just went to run a NN...Where is GeoXplore? Can make the input files, but can't do anything with them without GeoXplore to run the NN.

**Comment by trotor** [comment]
Do you mean the external .exe file ?

**Comment by tachyon-work** [comment]
Yes, but also why was the script that called the exe taken out of the NN menu?

The exe isn't in the SDM download package though, which is the main issue.

Sent from my phone

________________________________

**Comment by trotor** [comment]
I have to say I don't know or remember.
I only remember this SDMLR.exe file.
[image: image.png]

**Comment by tachyon-work** [comment]
The sdmlr.exe file is just to do the logistic regression.

There should be a second exe file named geoXplore.exe to run the neural networks. Without the exe, the entire neural networks menu in sdm is functionally useless.

Sent from my phone

________________________________

**Comment by trotor** [comment]
I added the geoexplore.exe to bin folder. All the other related code is missing and unfortunately I don't know when it can be done. Unless somebody.else@somewhere does it - I can add / give old code on request.
